### ISSUE TITLE
API Error

### ISSUE THREAD

**Issue by drummingdemon** [tuya/tuya-homebridge#243]
TuyaOpenAPI request: method = get, endpoint = https://openapi.tuyaeu.com, path = /v1.0/iot-01/associated-users/devices, params = {"size":100}, body = null, headers = {"t":"1652632019138","client_id":"ke89j4nysqfnrw8amkws","nonce":"d7531be0-d46b-11ec-a503-03a07257d441","Signature-Headers":"client_id","sign":"D1B928B0E86553B39AAF2E01E77F68D26EB4D14DD4AA497D6A71E860889F3032","sign_method":"HMAC-SHA256","access_token":"6b788b28b6ae49fe94b4d53411ad9490","lang":"en","dev_lang":"javascript","dev_channel":"homebridge","devVersion":"1.5.0"}

TuyaOpenAPI response: {"code":28841101,"msg":"No permissions. This API is not subscribed.","success":false,"t":1652632019297,"tid":"d80905a7d46b11ecafb0ea79ca261ada"} path = /v1.0/iot-01/associated-users/devices

Please advise, I've followed the instructions and even the API Exlorer shows an error...

**Comment by drummingdemon** [comment]
As a follow-up, to anyone else having this issue - I've opened a bug ticket within the `Tuya IoT Platform` dashboard and explained my issue - in fact, even the _API Explorer_ function told me I didn't authorize the services I tried to push a request to. Within a few hours I've got a generic response stating that I should enable the services - however, without me changing anything the API had requests suddenly gone through so I suspect someone changed something behind the scenes.

**TL;DR - Issue a bug ticket through the `Tuya IoT Platform` and API requests should go through**
